test/Microsoft.DotNet.ProjectJsonMigration.Tests/MSBuild.exe.config

```diff
@@ -0,0 +1 @@
+hey
```

Comment:
Are we sure we want to check in the `MSBuild.exe` file as well?  That seems less than ideal.

---
In reply to: [75883033](https://github.com/dotnet/cli/pull/4074#discussion_r75883033) [](ancestors = 75883033)